calculatorUICOOP/Models/Math_Logic.cs

```diff
@@ -33,6 +33,10 @@ public static double Divide(double num1, double num2)
         {
             return num1 / num2;
         }
+        public static double Module(double num1, double num2)
+        {
+            return num1 % num2;
+        }
```

Comment:
We're doing percent (num1 x 100), not the modulo operator.

Reply:
You mean division: num1 / 100 = 0.0{num1}